evm/src/executor/abi.rs

```diff
@@ -680,6 +681,291 @@ pub static HARDHAT_CONSOLE_SELECTOR_PATCHES: Lazy<HashMap<Selector, Selector>> =
     ])
 });
 
+enum FormatSpec {
```

Comment:
let's also move all of these to a spearate module (fmt.rs?)
can turn abi.rs in a directory module

Reply:
done